src/test/java/teammates/test/cases/browsertests/FeedbackConstSumOptionQuestionUiTest.java

```diff
@@ -204,4 +206,21 @@ public void testDeleteQuestionAction() {
         assertNull(BackDoor.getFeedbackQuestion(courseId, feedbackSessionName, 1));
     }
 
+    private void testUiConsistencyForNewQuestion() {
+        ______TS("CONSTSUM-option after CONSTSUM-recipient: Check ui consistency success case");
```

Comment:
Add an explanatory comment? Neither the method name nor the `TS` seems self-explanatory. 🤔